### PR TITLE
build!: update examples for v1

### PR DESCRIPTION
**Pull request by benelan**
Updates the examples for the v1 release.

BREAKING CHANGE: Removed Vue v2 and Ember v3 examples. Use Ve v3 and Ember v4 examples instead,

### CODE REVIEW

File: preact-typescript/README.md [hunk 1]
```diff
@@ -1,62 +1,19 @@
-# Preact
+# preact_new
```
Comment (geospatialem) [inline preact-typescript/README.md:1]: Should this be changed?

File: preact-typescript/src/components/header/style.css [hunk 2]
```diff
@@ -2,47 +2,50 @@
 	position: fixed;
 	left: 0;
 	top: 0;
+
```
Comment (geospatialem) [inline preact-typescript/src/components/header/style.css:5]: nitpick:

Is there an extra line showing here?

File: preact-typescript/src/components/header/style.css [hunk 3]
```diff
@@ -2,47 +2,50 @@
...
+	display: flex;
+	justify-content: space-between;
+
```
Comment (geospatialem) [inline preact-typescript/src/components/header/style.css:8]: nitpick:

Possibly another extra line here?

File: preact-typescript/src/components/header/style.css [hunk 4]
```diff
@@ -2,47 +2,50 @@
...
 	width: 100%;
-	height: 56px;
-	padding: 0;
-	background: #673AB7;
+	height: 3.5rem;
+
```
Comment (geospatialem) [inline preact-typescript/src/components/header/style.css:11]: nitpick:

One more extra line?

File: preact-typescript/src/routes/home/index.tsx [hunk 5]
```diff
@@ -1,22 +1,28 @@
...
+      <p>We even have support for TypeScript!</p>
+      <calcite-dropdown
+        oncalciteDropdownClose={(event) =>
+          console.log(
+            "closing dropdown, the following item(s) are selected",
+            (event.target as HTMLCalciteDropdownElement).selectedItems
+          )
+        }
+      >
+        <calcite-button icon-start="banana" slot="trigger">Sort</calcite-button>
```
Comment (geospatialem) [inline preact-typescript/src/routes/home/index.tsx:17]: 🧹

**Review by geospatialem — APPROVED** [review]
Few nitpicks for considerations, otherwise LGTM! 
🚀⭐

**Comment by eriklharper** [comment]
Curious why we removed old vue and ember examples?  I've definitely needed to use those in the past when debugging issues people reported that were specific to those versions.

**Comment by benelan** [comment]
It's too much overhead to maintain multiple versions of all the frameworks

**Pull request merged**